_includes/header.html

```diff
@@ -20,7 +20,11 @@
         </div>
         <div class="col-sm-19 col-md-20 margin-top-10" id=main-menu-wrapper>
           <div class="float-right hidden-xs" id=btn-call-for-action>
+            {% if site.links.download %}
+            <div id=btn-call-for-action><a href="{{ site.links.download }}" class="btn btn-huge btn-warning"><i class="fa fa-download"></i>Download</a></div>
+            {% else %}
             <div id=btn-call-for-action><a href=/downloads class="btn btn-huge btn-warning"><i class="fa fa-download"></i>Download</a></div>
```

Comment:
I think the default behavior should be to link to our eclipse.org/downloads page.

+1 for allowing the ability to change the URL!